src/methods.js

```diff
@@ -596,7 +596,17 @@ export default {
         named: params => set(params, 'privkeys', map(params.privkeys || [], () => '******'))
       }
     },
-    version: '>=0.7.0'
+    version: '>=0.7.0 <0.18.0'
+  },
+  signRawTransactionWithWallet: {
+    category: 'rawtransactions',
+    obfuscate: {
```

Comment:
Since there is no sensitive data on this method (see the method [signature]( https://github.com/bitcoin/bitcoin/blob/master/src/wallet/rpcwallet.cpp#L3243)), simply remove the `obfuscate` property.